frontend/src/routes.js

```diff
@@ -219,10 +217,11 @@ const routes = [
           }
         ]
       },
-      {
-        path: 'catalog',
-        element: <Catalog />
-      },
+      config.modules.datasets.active &&
```

Comment:
`&&` is wrong. The logic here: `if at least one of (dashboards, dashboards) is not present, catalog is not displayed` instead it should be `if all of them are not present, catalog is not displayed`